src/Sdk/Interfaces/RequestObjectInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
|`line 13`|
|---|
Please make sure that all the "getters" methods start with `getX()`.

Reply:
The reason why we shouldn't prefix it with `getXXX` is that once the DTO implement this interface, the serializer with serialize/deserialize the object by its getters and setters, in order to make no confusing, we don't prefix it with `getXXX`